test/linters/eslint.json

```diff
@@ -24,7 +24,7 @@
     "accessor-pairs": "error",
     "arrow-spacing": ["error", { "before": true, "after": true }],
     "block-spacing": ["error", "always"],
-    "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
+    "brace-style": "off",
```

Comment:
Why are you removing this rule ?

Reply:
I didn't touch this file, it's part of one of your commit https://github.com/kuzzleio/documentation-V2/commit/806ed83f1827749808ea4030adeb2fdf90aa828f